pt-BR/mail/chrome/messenger/msgHdrViewOverlay.dtd

```diff
@@ -76,15 +76,15 @@
 <!ENTITY copyLinkCmd.label        "Copiar link">
 <!ENTITY copyLinkCmd.accesskey    "o">
 
-<!ENTITY CopyMessageId.label "Copy Message-ID">
-<!ENTITY OpenMessageForMsgId.label "Open Message For ID">
-<!ENTITY OpenBrowserWithMsgId.label "Open Browser With Message-ID">
+<!ENTITY CopyMessageId.label "Copiar ID da mensagem">
+<!ENTITY OpenMessageForMsgId.label "Abrir mensagem pelo ID">
+<!ENTITY OpenBrowserWithMsgId.label "Abrir navegador com o ID da mensagem">
```

Comment:
@brunotiussi I'm not sure about lines 79 through 81.  I'm not sure where this is, so let's keep "Message-ID" and "ID" untranslated.